hystrix/src/main/java/feign/hystrix/HystrixFeign.java

```diff
@@ -0,0 +1,24 @@
+package feign.hystrix;
+
+import feign.Contract;
+import feign.Feign;
+
+public class HystrixFeign {
```

Comment:
final this and the builder? maybe needs javadoc saying that this allows your interfaces to return a HystrixCommand and decorates plain methods.